test/soca/create_obsdb.py

```diff
@@ -37,5 +37,34 @@
                                        'salt_profile_fnmoc',
                                        'icec_EMC',
                                        'icefb_GDR']})
+    ufsda.r2d2.store(obsstore)
 
+# Create the test R2D2 database for output from bufr2ioda tests
+    obsstore = NiceDict({'start': '2018-04-15T00:00:00Z',
+                         'end': '2018-04-15T00:00:00Z',
+                         'step': 'PT24H',
+                         'source_dir': '../../testoutput/',
+                         'source_file_fmt': '{source_dir}/{obs_type}_{year}{month}{day}.nc',
+                         'type': 'ob',
+                         'database': 'shared',
+                         'provider': 'gdasapp',
+                         'experiment': 'soca',
+                         'obs_types': ['bufr_dbuoyprof',
+                                       'bufr_mbuoybprof',
+                                       'bufr_sfcships',
+                                       'bufr_sfcshipsu']})
```

Comment:
@AndrewEichmann-NOAA, the bufr_tesacprof and bufr_trkobprof are missing.
You mentioned about dates and filename different. Can you elaborate on what you meant?

Reply:
bufr_tesacprof and bufr_trkobprof are handled separately, in the section below this. The ioda files here have the form` {obs_type}_{year}{month}{day}.nc`, like bufr_dbuoyprof_20180415.nc, but the other two are `{obs_type}_{year}{month}.nc` (missing the day), and as such have different entries for `source_file_fmt`. Hence the two calls to `ufsda.r2d2.store()`.